CMakeLists.txt

```diff
@@ -518,6 +520,40 @@ foreach(target IN LISTS targets)
     link_common_libraries(${target})
 endforeach()
 
+if (fuzzer_conditions)
```

Comment:
I don't think there is much need for a user-defined variable for this since you can just set it based on the `san` values, something like:  https://github.com/mellery451/rippled/commit/a303c0f78029016ae2069d46ad1570083b9d3ba7